Hamburg/HLC/RDW003.xml

```diff
@@ -0,0 +1,238 @@
+                     
+                       <!-- <layoutDesc>
+                           <layout columns="" writtenLines="">
+                              <locus from="" to=""/>
+                              <note corresp="#textarea1 #margin1">Data on text area and margin dimensions taken from <locus target="#33r"/>.</note>
+                              <note>Number of characters per line: </note>
+                                 <dimensions unit="mm" xml:id="textarea1">
+                              <height>113</height>
+                              <width>80</width>
+                           </dimensions>
+                           <dimensions type="margin" unit="mm" xml:id="margin1">
+                              <dim type="top">10</dim>
+                              <dim type="bottom">35</dim>
+                              <dim type="right">7</dim>
+                              <dim type="left">7</dim>
+                              <dim type="intercolumn">5</dim>
+                              <dim
+                           </dimensions>
+                              <ab type="ruling" subtype="pattern">Ruling pattern: </ab>
+                              <ab type="ruling"> </ab>
+                              <ab type="ruling"> </ab>
+                              <ab type="pricking"></ab>
+                              <ab type="pricking"></ab>
+                           </layout>
+                        </layoutDesc>-->
+                  </objectDesc>
+                  
+                  <!--    <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                            <date notBefore="" notAfter=""/>
+                           <desc></desc>
+                           <seg type="ink"></seg>
+                           <seg type="script"></seg>
+                           <seg type="rubrication"></seg>
+                        </handNote>
+                     </handDesc>-->
+                 
+                 <decoDesc>
+                    <decoNote xml:id="d1" type="band">
+                       <locus target="#3r"/>
+                       <desc>Ḥarag.</desc>
+                    </decoNote>
+                    
+                    <decoNote xml:id="d2" type="band">
+                       <locus target="#41r"/>
+                       <desc>Ḥarag.</desc>
+                    </decoNote>
+                 </decoDesc>
+                  
+                  
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="#1v"/>
+                           <desc type="GuestText">Short hymn.</desc>
+                           <q xml:lang="gez"/>
+                        </item>
+                        
+                        <item xml:id="a2">
+                           <locus target="#44v"/>
+                           <desc type="GuestText">Main text continued in a different hand.</desc>
```

Comment:
I am not sure about the category of an addition and Guest text here. It is not the guest text but the main one... Is to just switch of the hand?

Reply:
I actually do not understand the text (Mazmur ba-1 Yoḥannǝs) well enough right now. I will most probably anyway create an issue for its identification, maybe the identification of the end will also be helped then. For now, we could comment the addition and the type="complete" out.